src/renderer/components/dock/dock.store.ts

```diff
@@ -125,18 +127,18 @@ export class DockStore implements DockStorageState {
     }
   }
 
-  @action
+  @action.bound
```

Comment:
Why `bound`?

Reply:
It's just `@action` + autobind. 
Useful when destructing methods from object with keeping context of `this`:

```ts
const {toggleFillSize} = dockStore;
toggleFillSize(); // context of this == dockStore
```

Probably this should be avoided and called always directly, e.g. `dockStore.toggle()`